libraries/helpers_windows.rb

```diff
@@ -56,31 +58,35 @@ def build_rule(new_resource)
         type = to_type(new_resource)
         parameters = {}
 
-        parameters['description'] = "\"#{new_resource.description}\""
-        parameters['dir'] = new_resource.direction
-
-        new_resource.program && parameters['program'] = new_resource.program
-        parameters['service'] = new_resource.service ? new_resource.service : 'any'
-        parameters['protocol'] = new_resource.protocol
-
-        if new_resource.direction.to_sym == :out
-          parameters['localip'] = new_resource.source ? fixup_cidr(new_resource.source) : 'any'
-          parameters['localport'] = new_resource.source_port ? port_to_s(new_resource.source_port) : 'any'
-          parameters['interfacetype'] = new_resource.interface ? new_resource.interface : 'any'
-          parameters['remoteip'] = new_resource.destination ? fixup_cidr(new_resource.destination) : 'any'
-          parameters['remoteport'] = new_resource.dest_port ? port_to_s(new_resource.dest_port) : 'any'
+        if type == :log
+          "set currentprofile logging #{new_resource.logging} enable"
```

Comment:
This seems like an unexpected side effect of any rule that has logging. I don't think users will expect this to be run, if they haven't explicitly enabled logging.